microshift_configuring/microshift-kubeconfig.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+[id="microshift-cluster-access-kubeconfig"]
+= Cluster access with kubeconfig
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-networking
```

Comment:
The `:context:` must be the same as the `id`, and ideally they should also match the file name for the assembly. In this case, all three could be `microshift-cluster-access-kubeconfig` because that matches the assembly title.

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata